python/py-plyvel/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-plyvel
+version             1.0.4
+categories-append   devel
+platforms           darwin
+license             BSD
+
+python.versions     27 34 35 36
+
+maintainers         {ipglider.org:miguel @ipglider} openmaintainer
+
+description         Plyvel is a fast and feature-rich Python interface to LevelDB.
+long_description    Plyvel has a rich feature set, high performance, and a friendly Pythonic API.
```

Comment:
This `long_description` isn't very good, in that it gives us less information than `description` does. Remember these are used in automated systems, not just read in the file, so you need to include the fact that this is an interface to LevelDB in `long_description`.